### ISSUE TITLE
npm test appears to pass even when a spec file fails to build

### ISSUE THREAD

**Issue by adborden** [18F/cg-dashboard#938]
https://circleci.com/gh/18F/cg-dashboard/873

There is an error in [static_src/test/unit/actions/space_actions.spec.js](https://github.com/18F/cg-dashboard/blob/28c079b1989b94eaa542f9073db3a4ab4046883f/static_src/test/unit/actions/space_actions.spec.js) so the file fails to build but Karma still runs and reports success

```shell
# ...
webpack: bundle is now VALID.
Chrome 55.0.2883 (Linux 0.0.0): Executed 512 of 512 SUCCESS (0.893 secs / 0.782 secs)
```

The exit code is 0 locally, but fails on Circle CI with exit code 1 (thank goodness).

**Comment by msecret** [comment]
I've had to fix this before. I had to use the `WebpackKarmaWarningsPlugin` plugin. I only enabled it when `test` or `prod` are set for `NODE_ENV`. Maybe those aren't being set correctly.

**Comment by adborden** [comment]
Hmm, good catch. I thought `npm test` sets the `NODE_ENV=test`, but I guess not. In [circleci.yml](https://github.com/18F/cg-dashboard/blob/b6f50713d31ffa9494367d2e48635a686f1ad13f/circle.yml#L41) we are explicitly setting it. As part of our test setup, we can just set `NODE_ENV=test`.

**Comment by adborden** [comment]
This was fixed in d5d46213
